CanonicalLtd/documentation-builder
# Use documentation set logo as favicon

QA
--

``` bash
git clone https://github.com/canonicalltd/maas-docs
cd maas-docs
git clone -b add-favicons https://github.com/nottrobin/documentation-builder
python3 -m venv env3
env3/bin/pip install ./documentation-builder
env3/bin/python documentation-builder/bin/documentation-builder --force
```

Now open `build/en/index.html`, see that the favicon displays.

If not, maybe it's your cache, try a different browser with a clean cache.
